### PR TITLE
89 reevaluate charging curve

### LINKED ISSUE #89: Reevaluate charging curve
Restrucure and review code on charging curve. Also make charging curve accessible by config.

### PR DESCRIPTION
**Pull request by JakobWeg**
Charging curve is now specific for all car_types and can be configurated by points

### CODE REVIEW

File: simbev/car.py [hunk 1]
```diff
@@ -3,6 +3,7 @@
 import pandas as pd
 import pathlib
 import math
+from scipy.interpolate import interp1d
```
Comment (mosc5) [inline simbev/car.py:6]: Add scipy to requirements.txt

File: scenarios/default/configs/default.cfg [hunk 2]
```diff
@@ -60,6 +60,6 @@ hpc_data = hpc_config.csv
 # simulation parameters
 
 scaling = 1
-num_threads = 4
+num_threads = 1
```
Comment (mosc5) [inline scenarios/default/configs/default.cfg]: don't change the base scenario

File: simbev/helpers/helpers.py [hunk 3]
```diff
@@ -230,3 +226,10 @@ def timeit_wrapper(*args, **kwargs):
             return result
         return timeit_wrapper
     return decorator
+
+
+def interpolate_charging_curve(x, y):
+
+    f = interp1d(x, y, kind='cubic', fill_value='extrapolate')
+
+    return f
```
Comment (mosc5) [inline simbev/helpers/helpers.py:235]: Could we plot the full charging curves to show how they look => tells the user what we do with the input data

**Review by mosc5 — CHANGES_REQUESTED** [review]

File: simbev/helpers/helpers.py (hunk 3 above)
Comment (JakobWeg) [reply simbev/helpers/helpers.py:235]: Yep thats no problem. So you suggest an output of charging curves for every simulation?

File: simbev/car.py (hunk 1 above)
Comment (JakobWeg) [reply simbev/car.py:6]: Already done!

File: scenarios/default/configs/default.cfg (hunk 2 above)
Comment (JakobWeg) [reply scenarios/default/configs/default.cfg]: That was by accident.

File: simbev/helpers/helpers.py (hunk 3 above)
Comment (mosc5) [reply simbev/helpers/helpers.py:235]: For now we will just show the plots for out standard profiles in the documentation

**Review by mosc5 — APPROVED** [review]

**Pull request merged**